.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,46 @@
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+| Name  | about |title|labels|
+| ------------- | ------------- |------------- |------------- |
+| Bug Names  | Create a report to help us improve stellar-core |[Short Description] (Version: [stellar-core version])|bug|
+## Read before creating an issue
+In general, we close any issues that are
+*fill the template below under "Description"
+*questions best served elsewhere
+
+
+
+**Issue Description**
+Describe your bug/feature request in detail.
+
+**Steps to Reproduce**
+List in detail the exact steps to reproduce the unexpected behavior of the software.
+
+**Expected Result**
+Explain in detail what behavior you expected to happen.
+
+**Actual Result**
+Explain in detail what behavior actually happened.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+- OS: [e.g. iOS]
+- Browser [e.g. chrome, safari]
+- Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+- Device: [e.g. iPhone6]
+- OS: [e.g. iOS8.1]
+- Browser [e.g. stock browser, safari]
+- Version [e.g. 22]
+
+**Description**
```

Comment:
Can you revise the title as `Additional context` instead of `Description`?

Reply:
I cannot see any changes in this part. Can you revise it again?

Reply:
@Rapter1990 buradaki konu çözülmüşse, resolve edip mergeleyebiliriz.